code/modules/vehicles/mecha/combat/durand.dm

```diff
@@ -16,9 +16,9 @@
 	max_equip_by_category = list(
 		MECHA_L_ARM = 1,
 		MECHA_R_ARM = 1,
-		MECHA_UTILITY = 3,
+		MECHA_UTILITY = 1,
```

Comment:
All mechs should have a minimum of two utility slots. When I added mech thrusters as an equipment, air tanks were built into the mechs, but now it's an add-on. So you need two slots to make thrusters work.

Reply:
I'm not sure all need to have thrusters as an option, but from a QOL perspective it might be better just to lock the option on mechs that wouldn't be able to get it otherwise.